res/css/views/settings/tabs/_SettingsTab.pcss

```diff
@@ -62,6 +62,14 @@ limitations under the License.
     margin-inline-end: 80px; /* Align with the rest of the view */
     margin-bottom: 10px; /* TODO: Use a spacing variable */
     margin-inline-start: 0;
+
+    /* Making copy button stick to the right while scrolling */
+    .mx_CopyableText_border .mx_CopyableText_copyButton {
```

Comment:
It seems strange to put this under `mx_SettingsTab_subsectionText`. Could you explain why you feel it belongs here?

Reply:
Actually `.mx_CopyableText_copyButton` is used elsewhere too, so it would affect the button in other places too, to be more specific I did like this, as it's under `mx_SettingsTab_subsectionText` class. ( Checked with dev tools ). That's why.

Reply:
Firstly: it seems to me that these changes would be fine in the other places that CopyableText is used. Could you explain why you think they are not?

Secondly: even if we do want to limit changes to this specific usage, why `mx_SettingsTab_subsectionText` rather than next to your other changes?

Reply:
Oh, I see, I did make changes so that it won't affect other buttons having the same class, but now I see it more clearly, This can be achieved by writing the code in `_CopyableText.pcss` file, under the given selector `.mx_CopyableText .mx_CopyableText_copyButton`. We don't have to make changes explicitly in `mx_SettingsTab_subsectionText` code 😀, Thanks @richvdh ! I missed a huge point.